query/search.go

```diff
@@ -71,6 +71,20 @@ func SetupSearch(pathToTemplates string) (*template.Template, error) {
 	return templates, err
 }
 
+// SetupSearch loads templates for use by the search handler and should be done only once
```

Comment:
Function name in comment should be updated
```suggestion
// SetupV710Search loads v710 templates for use by the search handler and should be done only once
```